cob-pcs/cob-pcs-install.sh

```diff
@@ -9,16 +9,9 @@ if [ "$#" -lt 1 ]; then
 	exit 1
 fi
 
-robot_name="${HOSTNAME//-b1}"
-
-echo $robot_name
-client_list="
-$robot_name-b1
-$robot_name-t1
-$robot_name-t2
-$robot_name-t3
-$robot_name-s1
-$robot_name-h1"
+# get pcs in local network
+IP=$(`hostname -I | awk '{print $1}'`)
+client_list=$(nmap --unprivileged $IP-98 --system-dns | grep report | awk '{print $5}')
```

Comment:
please use ip addresses and not hostnames, the hostnames change depending on the network

`$(nmap --unprivileged $IP-98 --system-dns | grep report | awk '{print $6}' | sed 's/(//g;s/)//g')`